pkg/limayaml/defaults.go

```diff
@@ -390,6 +390,10 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		}
 	}
 
+	if y.MountType == "" {
+		y.MountType = "sshfs"
```

Comment:
Can we define constants for the mount types?

Reply:
Added REVSSHFS and NINEP (since 9P is not legal)